admin/onpage/class-onpage-email-presenter.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * This class is the presenter for the email. Based on the given parameters and data it will parse the email message
+ * that will be send
+ */
+class WPSEO_OnPage_Email_Presenter {
+
+	/**
+	 * @var string The generated messsage.
+	 */
+	private $message;
+
+	/**
+	 * @var array The old and the new status as an array.
+	 */
+	private $statuses;
+
+	/**
+	 * @var array The translated values for the indexability statuses.
+	 */
+	private $status_values;
+
+	/**
+	 * Constructing the object
+	 *
+	 * @param array $statuses Array with the statusses, normally it are the old and the new status.
```

Comment:
The plural of `status` is either [`statuses` or `status`](http://www.onlinegrammar.com.au/plural-of-status/).